astrokat/simulate.py

```diff
@@ -156,15 +156,20 @@ def _fake_slew_(self, target):
             if self.katpt_current is None:
                 slew_time = _DEFAULT_SLEW_TIME
             else:
-                user_logger.debug('DEBUG: slewing to {}'.format(target.name))
+                user_logger.debug('Slewing to {}'.format(target.name))
                 slew_time = self.slew_time(target)
         return slew_time
 
     def track(self, target, duration=0, announce=False):
         self.track_ = True
-        time.sleep(self._fake_slew_(target)+duration)
+        time.sleep(self._fake_slew_(target))
         now = timestamp2datetime(self.time)
         simobserver.date = ephem.Date(now)
+        user_logger.info('Slewed to {}'.format(target.name))
+        time.sleep(duration)
+        now = timestamp2datetime(self.time)
+        simobserver.date = ephem.Date(now)
+        user_logger.info('Tracked {} for {}'.format(target.name, duration))
```

Comment:
These two logs don't quite match what session outputs.  I thought the plan was for astrokat dry-run output to be as close as possible to what CAM will output.  See what session does [here](https://github.com/ska-sa/katcorelib/blob/bc4226fe8dcd433f8dfe0485d3cf0de7c436dbb7/katcorelib/mkat_session.py#L2383).

Reply:
We though session logging is too verbose. The messages left out did not add any timing information.